app/core/src/screens/hotels/AllHotelsNavigationScreen.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { TouchableOpacity } from 'react-native';
+import {
+  AllHotels,
+  type SearchParametersType,
+} from '@kiwicom/react-native-app-hotels';
+import { Ionicons } from '@expo/vector-icons';
```

Comment:
Note (no action required in this PR): Other parts of the app use MaterialIcons. We should make it consistent and choose one preferred set of icons.

Reply:
In this case icons from Ionicons and MaterialIcons are identical (the `md-` prefix on the Ionic's `md-map` stands for Material Design):

![screen shot 2018-01-03 at 11 16 27](https://user-images.githubusercontent.com/1473642/34516609-cb479eca-f077-11e7-8ab5-8e223352524a.png)

I think the design uses only Material Icons.

Reply:
Did you check all occurrences? If it's true we can change it easily. That would be great...

Reply:
Ah, you mean just this one case. Well, we can change it... @jaroslav-kubicek 🙂

Reply:
I didn't check all occurrences in the design, but in the code we currently use only this `md-map` from `Ionicons`. I will ask on Slack.

Reply:
Confirmed that all icons are from Material Design. https://skypicker.slack.com/archives/C800Q9ZQC/p1514976095000073?thread_ts=1514976043.000119&cid=C800Q9ZQC